airbyte-integrations/connectors/source-instagram/source_instagram/schemas/media_insights.json

```diff
@@ -33,10 +33,6 @@
       "description": "The number of times users have saved the media.",
       "type": ["null", "integer"]
     },
-    "video_views": {
```

Comment:
Probably not useful because the media_insights stream uses the schema inlined in the manifest but I updated both just in case